README.md

```diff
@@ -17,19 +17,19 @@ The Model Context Protocol allows applications to provide context for LLMs in a
 
 ## Installation
 
-Add the JitPack repository to your build file:
+Add the Space repository to your build file:
 
 ```kotlin
 repositories {
-    maven { url = uri("https://jitpack.io") }
+    maven { url = uri("https://maven.pkg.jetbrains.space/public/p/kotlin-mcp-sdk/sdk") }
 }
 ```
 
 Add the dependency:
 
 ```kotlin
 dependencies {
-    implementation("com.github.JetBrains:mcp-kotlin-sdk:main-SNAPSHOT")
+    implementation("com.github.JetBrains:mcp-kotlin-sdk:1.0-SNAPSHOT")
```

Comment:
Should we make 0.1.0 the first release? 1.0 please avoid using SNAPSHOT in the version